wiktionary_verb_scraper.py

```diff
@@ -0,0 +1,127 @@
+import argparse
+import csv
+import unicodedata
+from collections import defaultdict
+
+import requests
+from bs4 import BeautifulSoup
+
+
+def query_pages():
+    params = defaultdict(set, {
+        'format': 'json',
+        'action': 'query',
+        'list': 'categorymembers',
+        'cmtitle': 'Категория:Русские_глаголы',
+    })
+
+    while True:
+        resp = requests.get(
+            'https://ru.wiktionary.org/w/api.php',
+            params=params
```

Comment:
I used to create a copy of the params, probably I did that because some sort of bug appeared when not copying. Have you ensured everything continues to work after this change?

Reply:
Yes. Please check it correctly goes and parses the *next* page with words (this should trigger params change, if I can remember correctly) .

Reply:
Thanks for pointing that out! I think the PR can be merged as is then. I'll fix this later.